docs/general/staking-apps.md

```diff
@@ -1,17 +1,17 @@
 ---
-id: learn-staking-apps
+id: staking-apps
 title: Polkadot Staking Apps
 sidebar_label: Staking Apps
 description: Staking Apps in the Polkadot Ecosystem.
 keywords: [staking, stake, nominate, nominating, NPoS, staking dashboard]
-slug: ../learn-staking-apps
+slug: ../staking-apps
```

Comment:
This will break the Wiki link in your tweet this morning. Perhaps, keep it as is?

Reply:
I will create a new tweet today after merging and pushing to prod, specifying the two pages and who should contribute to "Staking Apps" and "Parachains Apps" :)